.travis.yml

```diff
@@ -39,6 +44,8 @@ before_install:
     else
       export HOST_OS="$TRAVIS_OS_NAME";
     fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install ccache; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then PATH=/usr/local/opt/ccache/libexec:$PATH; fi
```

Comment:
Fair enough. Why is this needed though? Shouldn't `brew` take care of symlinking any binary to `/usr/local/bin`?